clouddriver-ecs/src/test/groovy/com/netflix/spinnaker/clouddriver/ecs/provider/view/EcsLoadBalancerProviderSpec.groovy

```diff
@@ -16,33 +16,53 @@
 
 package com.netflix.spinnaker.clouddriver.ecs.provider.view
 
+import com.amazonaws.services.ecs.model.LoadBalancer
 import com.netflix.spinnaker.clouddriver.ecs.EcsCloudProvider
 import com.netflix.spinnaker.clouddriver.ecs.cache.client.EcsLoadbalancerCacheClient
+import com.netflix.spinnaker.clouddriver.ecs.cache.client.EcsTargetGroupCacheClient
+import com.netflix.spinnaker.clouddriver.ecs.cache.client.ServiceCacheClient
 import com.netflix.spinnaker.clouddriver.ecs.cache.model.EcsLoadBalancerCache
+import com.netflix.spinnaker.clouddriver.ecs.cache.model.Service
+import com.netflix.spinnaker.clouddriver.ecs.model.loadbalancer.EcsLoadBalancer
+import com.netflix.spinnaker.clouddriver.ecs.model.loadbalancer.EcsTargetGroup
 import com.netflix.spinnaker.clouddriver.ecs.security.ECSCredentialsConfig
+import com.netflix.spinnaker.clouddriver.ecs.security.NetflixECSCredentials
 import spock.lang.Specification
 import spock.lang.Subject
 
 class EcsLoadBalancerProviderSpec extends Specification {
-  def client = Mock(EcsLoadbalancerCacheClient)
-  def ecsCredentialsConfig = Mock(ECSCredentialsConfig)
+  public static final String ECS_ACCOUNT = 'ecsAccount'
+  public static final String AWS_ACCOUNT = 'awsAccount'
```

Comment:
nit: you might want to stick to using def or defining concrete types in both test classes (just for consistency). In EcsLoadBalancerCacheClientSpec you use def, where as here you use the concrete type.